packages/calendar/src/components/input-types/date-range-input/DateRangeInput.tsx

```diff
@@ -157,6 +157,7 @@ export const DateRangeInput: React.FC<DateRangeInputProps> = ({
               borderColor={colors.borderColor}
               indent
               spacing
+              shadow={"box"}
```

Comment:
Mistake or correct?

Reply:
Hm, maybe we should have a third shadow for popover stuff?
Which could look the same, for now, but it might be important to separate them semantically.

Reply:
Is DateRangInput using a modal or why is this in this PR? Lazy? =) And yeah, that could be nince to have a third shadow that is the same as modal

Reply:
I added a `popover` shadow. It is in between `modal` and `box`. Ozcar wants more shadows later, but that will have to be a separate discussion and PR :)